### PR TITLE
Add EIP: Elected block proposer has not been slashed

### PR DESCRIPTION
**Pull request by mkalinin**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: EIPS/eip-6987.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+eip: 6987
```
Comment (abcoathup) [inline EIPS/eip-6987.md:2]:
```suggestion
eip: 6988
```
EIP numbers are assigned by EIP editors and are generally the PR number which would make this EIP6988.  There isn't an issue or PR 6987 that I can see.

File: EIPS/eip-6987.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+discussions-to: <URL>
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-05-04
+---
+
+## Abstract
+
+Introduces a modificaiton to the consensus layer specification which ensures that slashed validator cannot be elected as block proposer.
```
Comment (abcoathup) [inline EIPS/eip-6987.md:15]:
```suggestion
Introduces a modification to the consensus layer specification which ensures that slashed validator cannot be elected as block proposer.
```
Typo

File: EIPS/eip-6987.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+## Motivation
+
+A block proposed by a slashed validator is invalidated by the corresponding check in the [`phase0/process_block_header`](https://github.com/ethereum/consensus-specs/blob/3115d1140b23dd4c9c23fbd9e2428186cf816bde/specs/phase0/beacon-chain.md#block-header) function as it is defined in the consensus layer specification.
```
Comment (g11tech) [inline EIPS/eip-6987.md:19]:
```suggestion
A block proposed by a slashed validator is rejected by a corresponding validation check in the [`phase0/process_block_header`](https://github.com/ethereum/consensus-specs/blob/3115d1140b23dd4c9c23fbd9e2428186cf816bde/specs/phase0/beacon-chain.md#block-header) function as it is defined in the consensus layer specification.
```

File: EIPS/eip-6987.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+At the same time the definition of the [`phase0/compute_proposer_index`](https://github.com/ethereum/consensus-specs/blob/3115d1140b23dd4c9c23fbd9e2428186cf816bde/specs/phase0/beacon-chain.md#compute_proposer_index) allows for a slashed validator to be elected as a proposer. This contradiction may effectively lead to an empty slot if a block is supposed to be proposed by a slahed validator.
```
Comment (g11tech) [inline EIPS/eip-6987.md:21]:
```suggestion
At the same time the definition of the [`phase0/compute_proposer_index`](https://github.com/ethereum/consensus-specs/blob/3115d1140b23dd4c9c23fbd9e2428186cf816bde/specs/phase0/beacon-chain.md#compute_proposer_index) allows for a slashed validator to be elected as a proposer. This contradiction effectively leads to an empty slot if a block is supposed to be proposed by a slashed validator.
```

File: EIPS/eip-6987.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+An impact of the proposed fix in the case of a single slashing on Ethereum Mainnet is negligible, while it becomes significant in the case of correlated slashings. For instance, a correlated slashing of `1/10th` of a validator set can lead to `1/10th` of missed proposals during a number of epochs after the slashing.
+
+## Specification
+
+The proposed change is specified in [the consensus layer specification](https://github.com/ethereum/consensus-specs/blob/82cd2f1f2317f664b7e179acdd898a327e1fe77e/specs/deneb/beacon-chain.md#modified-compute_proposer_index).
+
+_TODO: move specification to `/_features` and provide a link._
```
Comment (g11tech) [inline EIPS/eip-6987.md:29]: noting TODO for tracking purposes

**Pull request merged**